### PR TITLE
Fix links in project structure section

### PR DESCRIPTION
**Pull request by jdanekrh**

### CODE REVIEW

**Comment by jdanekrh** [comment]
I signed it

File: README.md [hunk 1]
```diff
@@ -11,12 +11,11 @@ This project contains the following top level components:
 
 * [api](api/): The OpenTelemetry API.
   * [trace](api/src/main/java/io/opentelemetry/trace/): The tracing api. Includes `Tracer`, `Span` and `SpanContext`.
-  * [tags](api/src/main/java/io/opentelemetry/tags/): Key-value pairs of data that can be propagated to provide contextual information.
+  * [distributedcontext](api/src/main/java/io/opentelemetry/distributedcontext/): Collection of tags in the form of key-value pairs of data that can be propagated to provide contextual information.
```
Comment (carlosalberto) [inline README.md]: Collection of entries? (to avoid confusion with `tags`, which was previously used).

**Comment by carlosalberto** [comment]
Hey! Left a minor suggestion, thanks for the PR!

File: README.md (hunk 1 above)
Comment (jdanekrh) [reply README.md]: All right, I made the change.

**Comment by bogdandrutu** [comment]
Rebase to pickup the travis fix?

**Pull request merged**